index.html

```diff
@@ -188,8 +188,38 @@ <h3 id="h3">LET'S CONNECT</h3>
                     </ul>
                 </div>
             </section>
-
         </main>
+
+        <footer>
+            <section class="contact-form">
+                <h1 id="contact-me">Contact me </h1>
+                <p id="contact-text">
+                    If you ave an application you are interested in developing, a feature that you
+                    need built or a project that needs coding. I'd love to help with it.
+                </p>
+                <form>
```

Comment:
- [ ] [REQUIRED] Kindly, make the Form uses the `action` and `method` attribute to integrate with [formspree service](https://formspree.io/html) 🔗.

Reply:
Good job in implementing the changes 👍